common/script/ops/addWebhook.js

```diff
@@ -10,7 +10,7 @@ module.exports = function addWebhook (user, req = {}) {
   let wh = user.preferences.webhooks;
 
   if (!validator.isURL(_.get(req, 'body.url'))) throw new BadRequest(i18n.t('invalidUrl', req.language));
-  if (!validator.isBoolean(_.get(req, 'body.enabled'))) throw new BadRequest(i18n.t('invalidEnabled', req.language));
+  if (!validator.isBoolean(String(_.get(req, 'body.enabled')))) throw new BadRequest(i18n.t('invalidEnabled', req.language));
```

Comment:
Validate.js has deprecated testing anything but strings (it warned me while
testing), so to be preemptive I changed it before it became an error.
On Tue, Jun 7, 2016 at 10:03 PM Blade Barringer <notifications@github.com>
wrote:

> In common/script/ops/addWebhook.js
> <https://github.com/HabitRPG/habitrpg/pull/7622#discussion_r66186362>:
>
> > @@ -10,7 +10,7 @@ module.exports = function addWebhook (user, req = {}) {
> >    let wh = user.preferences.webhooks;
> >
> >    if (!validator.isURL(_.get(req, 'body.url'))) throw new BadRequest(i18n.t('invalidUrl', req.language));
> > -  if (!validator.isBoolean(_.get(req, 'body.enabled'))) throw new BadRequest(i18n.t('invalidEnabled', req.language));
> > +  if (!validator.isBoolean(String(_.get(req, 'body.enabled')))) throw new BadRequest(i18n.t('invalidEnabled', req.language));
>
> Why are you wrapping this in a string object?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7622/files/e7245138d1fb1ef939f0e195762c2b6075117013#r66186362>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALz_81kMN9P4pKXfl7yRpZeO2fMRNZKxks5qJjD1gaJpZM4Iwhoh>
> .
>